duplicates.php

```diff
@@ -104,24 +104,21 @@ function processInput($input, $delimiter)
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <title> ConsoliDev | Duplicates </title>
-    <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8"/>
+    <title>ConsoliDev | Duplicates </title>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
     <link rel="stylesheet" href="CSS/styles.css"/>
-    <style>
-        /* Basic styling for the two columns */
-        .container {
-            display: flex;
-            max-width: 80vw;
-            margin: auto;
-            gap: 20px;
-        }
-
-        .leftHalf, .rightHalf {
-            width: 50%; /* Split the width equally */
-        }
-    </style>
+    <link rel="stylesheet" href="CSS/duplicates.css"/>
+    <script src="https://kit.fontawesome.com/d0af7889fc.js" crossorigin="anonymous"></script>
 </head>
+
 <body>
+<?php include('header.php'); ?>
+<main class="content-wrapper">
+
+<div class="title-container">
+    <i class="fa-solid fa-magnifying-glass title-icon"></i>
+	<h1 class="page-title">Duplicate Finder/Remover</h1>
+<div>
 
 <div class="container">
```

Comment:
Whenever you resize the boxes it cuts into the other one. 
![image](https://github.com/user-attachments/assets/4ac7eb0d-9d20-482a-9e70-701626f21695)

Reply:
nahhh not really